### PR TITLE
Add application icons

### LINKED ISSUE #99: Provide an application icon

### PR DESCRIPTION
**Pull request by allaeddineomc**
Resolves #99

### CODE REVIEW

File: src/application.rs [hunk 1]
```diff
@@ -146,7 +146,7 @@ impl Application {
             .translator_credits(&gettext("translator-credits"))
             .modal(true)
             .authors(vec!["Marcus Behrendt".into()])
-            .artists(vec!["Marcus Behrendt".into()])
+            .artists(vec!["Marcus Behrendt".into(), "Allaeddine Boulefaat".into()])
```
Comment (marhkb) [inline src/application.rs:149]: rustfmt complains. This should fix it.
```suggestion
            .artists(vec![
                "Marcus Behrendt".into(),
                "Allaeddine Boulefaat".into(),
            ])
```

**Review by marhkb — CHANGES_REQUESTED** [review]

**Comment by marhkb** [comment]
There is one thing I noticed: The icon appears a bit too small. Could you make it the same height and width like nautilus?

comparison

**Comment by marhkb** [comment]
Thanks. I noticed there's a problem with the symbolic icon. It's rendered dark instead of white. Can you  confirm?

Bildschirmfoto vom 2022-05-03 22-40-04
Bildschirmfoto vom 2022-05-03 22-41-16

**Comment by allaeddineomc** [comment]
> Thanks. I noticed there's a problem with the symbolic icon. It's rendered dark instead of white. Can you confirm?
> 
> Bildschirmfoto vom 2022-05-03 22-40-04 Bildschirmfoto vom 2022-05-03 22-41-16

yes i just noticed , i have no idea how to fix this , it renders correctly inside app icon preview but it's black when exported

**Comment by marhkb** [comment]
I think I know why.

There is a general problem with the icons itself. Inside the svg you seem to import a png, which is not great because it may render them blurry and GNOME Shell cannot recolor the icons. 

The icons should only consist of geometry elements.

**Comment by allaeddineomc** [comment]
so that's an issue with app icon preview app itself , the symbolic icon is vector in the source SVG but it's being exported as a PNG when using app icon preview , i will make it to it's own SVG file manually

**Comment by allaeddineomc** [comment]
Screenshot from 2022-05-03 23-20-20

**Comment by marhkb** [comment]
Yes, that looks good. The other icons need to be changed, too. They also have an embedded pngs.

**Comment by marhkb** [comment]
It's good now. At last, for merging, I need you to squash all the 5 commits into a single commit.

**Comment by allaeddineomc** [comment]
done

**Pull request merged**